services/serverless.yaml

```diff
@@ -35,6 +35,7 @@ functions:
           path: publish
           method: post
           cors: true
+      - schedule: cron(0 1 * * ? *)
```

Comment:
This will enable scheduling to invoke publish lambda on 1am of every day. We need to keep an eye that this is actually happening.